spec/order_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'order'
+
+describe Order do
+  it 'selects food from menu' do
+    order = Order.new
+    expect(order.current_selected_food).to eq nil
+  end
+  it 'puts food into ordered food array' do
+    order = Order.new
```

Comment:
Good RSpec, maybe also have a test where you check items are going into the array.